.github/workflows/ci.yml

```
@@ -109,3 +112,61 @@ jobs:
       # # Nx Affected command for integration tests
       - name: Run affected for integration tests
         run: npx nx run-many -t=build --projects@tailwindcss/standalone && npx nx affected -t=postbuild && npx nx affected -t=integration-test --parallel=3
+
+
+  macos-dte-agents:
+    name: MacOS DTE Agent
+    runs-on: macos-14
+    timeout-minutes: 30
+
+    env:
+      NX_CI_EXECUTION_ENV: macos
+      NX_BRANCH: ${{ github.event.pull_request.head.ref || github.ref }}
+
+    strategy:
+      matrix:
+        node-version: [20]
+        agent: [1, 2, 3]
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v4
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'pnpm'
+
+      # Cache cargo if Rust
+      - name: Cache cargo
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      # Install Rust
+      - name: Install Rust (Stable)
+        run: rustup default stable
+
+      - name: Setup rust target
+        run: rustup target add ${{ matrix.target }}
```

```suggestion
      - name: Setup rust target
        run: rustup target add aarch64-apple-darwin
```

If we are running this on MacOS is this needed?